config/sql/annotations.sql

```diff
@@ -2,11 +2,11 @@
 
 -- DROP TABLE public.annotations;
 
-CREATE TABLE public.annotations
+CREATE TABLE IF NOT EXISTS public.annotations
 (
   id text NOT NULL,
   annotations xml,
   CONSTRAINT annotations_id_key UNIQUE (id)
 );
 
-GRANT ALL ON TABLE public.annotations TO kemal;
+GRANT ALL ON TABLE public.annotations TO current_user;
```

Comment:
Using `current_user` so we no longer hardcode the username to `kemal` and will take whatever's provided by the config or environment variable.